hypothesis-python/tests/array_api/README.md

```diff
@@ -48,3 +48,6 @@ Otherwise the test suite will use the variable as the `api_version` argument for
 In the future we intend to support running tests against multiple API versioned
 namespaces, likely with an additional recognized option that infers all
 supported versions.
+
+[^1]: Note NumPy will likely remove `numpy.array_api` in the future ([NEP 56](https://github.com/numpy/numpy/pull/25542))
```

Comment:
I'd prefer a link to https://numpy.org/neps/nep-0056-array-api-main-namespace.html